recipes/urduhack/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "urduhack" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 66147eb1eb602cf0d63bf832472983f1bd514b58b5ae61b0cee3aa1f12f02631
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - pip
+    - python
```

This shouldn't be needed.
```suggestion
  build:
    - pip
    - python
```